data_analysis_pipeline/02_word2vec_doc_trainer.py

```diff
@@ -0,0 +1,61 @@
+import argparse
+from pathlib import Path
+import multiprocessing
+import re
+
+from gensim.models import Word2Vec
+import numpy as np
+import tqdm
+
+from word2vec_timeline_utils import train_word2vec, SentenceIterator
+
+
+parser = argparse.ArgumentParser(description="Train multiple Word2Vec models")
+parser.add_argument(
+    "--parse_file_path", help="The file that contains every sentence per line,"
+)
+parser.add_argument(
+    "--model_cat",
+    help="The category to label each model output. Typically it's (fulltext or abstract)",
```

Comment:
Should this have a `choices` parameter?

Reply:
Actually, this should be removed as the project only uses full text now. I removed it.